### PR TITLE
Merge insert_size_hist into main

### PR DESCRIPTION
**Pull request by mei-knudson**
Updated code to produce ATAC insert size histogram plot as png

### CODE REVIEW

File: tasks/share_task_qc_atac.wdl [hunk 1]
```diff
@@ -36,6 +34,7 @@ task qc_atac {
     String stats_log = '${default="share-seq" prefix}.atac.qc.stats.${genome_name}.log.txt'
     String hist_log = '${default="share-seq" prefix}.atac.qc.hist.${genome_name}.log.txt'
     String hist_log_pdf = '${default="share-seq" prefix}.atac.qc.hist.${genome_name}.log.pdf'
```
Comment (emattei) [inline tasks/share_task_qc_atac.wdl:36]: This string is not required anymore

**Review by emattei — CHANGES_REQUESTED** [review]

File: tasks/share_task_qc_atac.wdl (hunk 1 above)
Comment (mei-knudson) [reply tasks/share_task_qc_atac.wdl:36]: See comment on commit 24e52e8; the name of the pdf is a required input to picard CollectInsertSizeMetrics

Comment (emattei) [reply tasks/share_task_qc_atac.wdl:36]: Perfect,  then please add  the comment " the name of the pdf is a required input to picard CollectInsertSizeMetrics" so even in the future we will know why that string is required even if we are not using it and we don't want a pdf.

Comment (mei-knudson) [reply tasks/share_task_qc_atac.wdl:36]: My pleasure

**Review by emattei — APPROVED** [review]

**Comment by emattei** [comment]
Everything should be fine. Eugenio needs to make sure the docker is build correctly like intended.

**Pull request merged**